bazelisk.py

```diff
@@ -314,7 +314,21 @@ def download_bazel_into_directory(version, is_commit, directory):
                 sys.stderr.write(
                     "The Bazel mirror does not have a checksum file; skipping checksum verification."
                 )
-                return destination_path
+                if "https://releases.bazel.build" not in bazel_url:
+                    matched=re.match(r"(\d*\.\d*(?:\.\d*)?)(rc\d+)?", version)
+                    if matched:
+                        (version, rc) = matched.groups()
+                        fallback_url="https://releases.bazel.build/{}/{}/{}".format(
+                            version, rc if rc else "release", bazel_filename
+                        )
+                        download(fallback_url+".sha256", sha256_path)
```

Comment:
Let's be consistent and use `format` here for string concatenation
```suggestion
                        download("{}.sha256".format(fallback_url), sha256_path)
```